claudia-lab/app/view/profile/profile.scss

```diff
@@ -0,0 +1,8 @@
+@import "theme";
+
+.profile {
+  width: 100%;
+  height: 20%;
+
+  background-color: $brand-white;
+}
```

Comment:
@loomnugget since your profile and home CSS are the same, you could create a partial _or_ a mixin to share styles - for more info on mixins:  https://www.sitepoint.com/sass-basics-the-mixin-directive/